discovery_engine_core/core/src/stack/ops/breaking.rs

```diff
@@ -70,10 +69,15 @@ impl BreakingNews {
     }
 }
 
+/// 1ce442c8-8a96-433e-91db-c0bee37e5a83 (valid uuid)
+pub(crate) const BREAKING_NEWS_ID: Id = Id::from_bytes([
```

Comment:
We don't need this, hopefully we will never have to inspect the id of the stack after creation and when uuid 1.0 will be released we can use `uuid!` that will parse the string and create the uuid at compile time. At the moment there is an alpha version.

Reply:
i'll revert this, it's a leftover of the first impl iteration.